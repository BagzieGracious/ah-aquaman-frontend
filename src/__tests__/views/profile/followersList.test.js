import { shallow } from "enzyme";
import React from "react";
import FollowersList from "../../../views/profile/followersList";

describe("followers List component tests", () => {
  it("should render without crashing", () => {
    const props = {
      followersList: [
        { image: "test.jpg", username: "truly", bio: "am cool" },
      ],
    };
    const wrapper = shallow(<FollowersList {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
