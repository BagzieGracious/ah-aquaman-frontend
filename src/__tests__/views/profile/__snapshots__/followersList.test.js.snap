// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`followers List component tests should render without crashing 1`] = `ShallowWrapper {}`;
